.github/templates/jobs/test.erb

```diff
@@ -0,0 +1,11 @@
+test:
+  name: Test
+  runs-on: ubuntu-latest
+
+  strategy:
+    matrix:
+      ruby: [2.4.x, 2.5.x, 2.6.x, 2.7.x]
```

Comment:
Again, should this be hardcoded?

Reply:
I don't think it needs templating because it's only specified in one place, and templating it just adds to the indirection and hides the source of this somewhere else.